Python/ibmcloudsql/SQLQuery.py

```diff
@@ -476,7 +476,7 @@ def get_result(self, jobId, pagenumber=None):
         Parameters
         ----------
         jobId: int
-            The value, if not stored, can be retrieved from self.get_jobs()
+            The value, if not stored, can be retrieved from :meth:`.get_jobs`
```

Comment:
needs `:py` before `:meth:`?

Reply:
If documenting Python code, `:py` is used by default if nothing before `:meth:`